test/problems/bndqp.jl

```diff
@@ -18,3 +18,16 @@ function bndqp_QP()
 
   return QuadraticModel(c, H, lvar=lvar, uvar=uvar, x0=x0, name="bndqp_QP")
 end
+
+function bndqp_QPSData()
+  c    = [1.0; 1.0]
+  H    = [-2.0 0.0; 3.0 4.0]
+  uvar = [1.0; 1.0]
+  lvar = [0.0; 0.0]
+  x0   = [0.5; 0.5]
+  qps = QPSData()
```

Comment:
Use the constructor to define a complete object:
```
qps = QPSData(2, 0, :min, 0.0, c, findnz(sparse(H))...,
    Int[], Int[], Float64[], Float64[], Float64[], lvar, uvar, "bndqp",
    nothing, nothing, nothing, nothing, nothing, String[], String[],
    Dict{String, Int}(), Dict{String, Int}(),
    fill(QPSReader.VTYPE_Continuous, 2), [QPSReader.RTYPE_Objective])
```

Reply:
This doesn't work. Maybe the 0 argument constructor of QPSData has erased the default constructor? It may be possible to add another constructor to the QPSData mutable struct, but perhaps it is easier to leave it like this?